src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+import { Button } from '../Button/Button';
+import { Icon } from '../Icon/Icon';
+import { Spinner } from '../Spinner/Spinner';
+
+const ALERT_TYPE_TO_ICON_MAPPING: { [type: string]: string } = {
+	info: 'circle-info',
+	success: 'circle-check',
+	danger: 'circle-warning',
+};
+
+export interface AlertProps {
+	close: () => void;
+	dark?: boolean;
+	message: ReactNode | string;
+	type?: 'info' | 'success' | 'danger' | 'spinner';
+}
+
+export const Alert: FunctionComponent<AlertProps> = ({
+	close = () => {},
+	dark = false,
+	message,
+	type = 'info',
+}) => {
+	return (
+		<div className={classNames('c-alert', { 'c-alert--dark': dark })}>
```

Comment:
we should fix some of the styles of mono:
![image](https://user-images.githubusercontent.com/1710840/61531443-17211000-aa27-11e9-936c-1f5d0b64ebe7.png)

Reply:
Add it on the backlog? It's not broken so shouldn't be added here imo.